infrastructure/modules/container-apps/main.tf

```diff
@@ -64,14 +100,33 @@ module "webapp" {
   app_key_vault_id                 = var.app_key_vault_id
   docker_image                     = var.docker_image
   user_assigned_identity_ids       = [module.db_connect_identity.id]
+  environment_variables            = local.webapp_env_vars
+  is_web_app                       = true
+  port                             = 8000
+}
+
+
+module "webapp_database" {
+  providers = {
+    azurerm     = azurerm
+    azurerm.hub = azurerm.hub
+  }
+  source                           = "../dtos-devops-templates/infrastructure/modules/container-app"
+  name                             = "${var.app_short_name}-db-${var.environment}"
+  container_app_environment_id     = var.container_app_environment_id
+  resource_group_name              = azurerm_resource_group.main.name
+  fetch_secrets_from_app_key_vault = var.fetch_secrets_from_app_key_vault
+  infra_key_vault_name             = "kv-${var.app_short_name}-${var.env_config}-inf"
+  infra_key_vault_rg               = "rg-${var.app_short_name}-${var.env_config}-infra"
+  enable_auth                      = false
+  app_key_vault_id                 = var.app_key_vault_id
+  docker_image                     = "postgres:16"
+  is_tcp_app                       = true
   environment_variables = {
-    ALLOWED_HOSTS   = "${var.app_short_name}-web-${var.environment}.${var.default_domain}"
-    DATABASE_HOST   = module.postgres.host
-    DATABASE_NAME   = module.postgres.database_names[0]
-    DATABASE_USER   = module.db_connect_identity.name
-    SSL_MODE        = "require"
-    AZURE_CLIENT_ID = module.db_connect_identity.client_id
+    POSTGRES_PASSWORD         = "secret"
```

Comment:
Can this be randomly generated? Such as https://github.com/NHSDigital/dtos-devops-templates/blob/main/infrastructure/modules/postgresql-flexible/main.tf#L48